### DISCUSSION TITLE
[Fast Data] Error when committing system of records deletion

### DISCUSSION THREAD

**Discussion by calairoh** [mia-platform/community#30] [Q&A]
Hi,
we're trying to remove an old System of Records from Fast Data Projections section. When committing the system of record deletion we get the following error:

![image](https://user-images.githubusercontent.com/31769199/191275735-449cd18d-7320-4bba-848a-86d6c11e2a12.png)

**Comment by danibix95** [comment]
Hi,

could you please provide the `version` set in the `fastdata-config.json` file, which can be found in the root folder of your project configuration repository?
This would help us understand whether the schema of Fast Data configuration file still matches the expected one.

**Reply by calairoh** [reply]
The `version` is `0.2.1`.
